src/bundle/Controller/ContentViewController.php

```diff
@@ -0,0 +1,94 @@
+ */
+
+namespace EzSystems\HybridPlatformUiBundle\Controller;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+use eZ\Publish\Core\MVC\Symfony\Controller\Controller;
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+
+class ContentViewController extends Controller
+{
+    protected $defaultSortFields = [
+        Location::SORT_FIELD_NAME => 'sort.name',
+        Location::SORT_FIELD_PRIORITY => 'sort.priority',
+        Location::SORT_FIELD_MODIFIED => 'sort.modified',
+        Location::SORT_FIELD_PUBLISHED => 'sort.published'
+    ];
+
+    protected $otherSortFields = [
+        Location::SORT_FIELD_PATH => 'sort.path',
+        Location::SORT_FIELD_CLASS_IDENTIFIER => 'sort.content.type.identifier',
+        Location::SORT_FIELD_SECTION => 'sort.section',
+        Location::SORT_FIELD_DEPTH => 'sort.depth',
+        Location::SORT_FIELD_CLASS_NAME => 'sort.content.type.name',
+    ];
+
+    public function detailsTabAction(ContentView $view)
+    {
+        $versionInfo = $view->getContent()->getVersionInfo();
+        $contentInfo = $versionInfo->getContentInfo();
+
+        $sectionService = $this->getRepository()->getSectionService();
+        $section = $sectionService->loadSection($contentInfo->sectionId);
+
+        $userService = $this->getRepository()->getUserService();
+        $creator = $userService->loadUser($contentInfo->ownerId);
+        $lastContributor = $userService->loadUser($versionInfo->creatorId);
+
+        $location = $view->getLocation();
+
+        $view->addParameters([
+            'section' => $section,
+            'contentInfo' => $contentInfo,
+            'versionInfo' => $versionInfo,
+            'creator' => $creator,
+            'lastContributor' => $lastContributor,
+            'translations' => $this->getTranslations($versionInfo),
+            'ordering' => [
+                'sortFields' => $this->getSortFields($location->sortField),
+                'sortOrders' => $this->getSortOrders()
+            ]
+        ]);
+
+        return $view;
+    }
+
+    protected function getTranslations(VersionInfo $versionInfo)
+    {
+        $languageRepository = $this->getRepository()->getContentLanguageService();
+
+        $translations = [];
+        foreach ($versionInfo->languageCodes as $languageCode) {
+            $translations[] = $languageRepository->loadLanguage($languageCode);
+        }
+
+        return $translations;
+    }
+
+    protected function getSortFields($currentSortField)
+    {
+        $sortFields = $this->defaultSortFields;
+
+        if (array_key_exists($currentSortField, $this->otherSortFields)) {
+            $sortFields[$currentSortField] = $this->otherSortFields[$currentSortField];
+        }
+
+        return $sortFields;
+    }
+
+    protected function getSortOrders()
+    {
+        return [
+            Location::SORT_ORDER_ASC => 'locationview.details.ascending',
```

Comment:
Need to check how we handled those in v1, @dpobel.

There are tricks for some of them, @mcurran-inviqa. Not every string is as easy to extract. For now it will work, as the platform ui v1 strings are defined. But we need to make sure that the extractor knows about those somehow.

Note that I have created a story for i18n: https://jira.ez.no/browse/EZP-27494.